docs/website/scripts/integration-index/integrations.yaml

```diff
@@ -0,0 +1,522 @@
+    - styra
+    - docker
+    - snyk
+  
+  iptables:
+    title: IPTables
+    description: IPTables is a useful tool available to Linux kernel for filtering network packets. OPA makes it possible to manage IPTables rules using context-aware policy.
+    labels:
+      layer: network
+      category: linux
+    software:
+    - Linux
+    tutorials:
+    - https://github.com/open-policy-agent/contrib/blob/master/opa-iptables/docs/tutorial.md
+    code: 
+    - https://github.com/open-policy-agent/contrib/tree/master/opa-iptables
+    inventors:
+    - name: Urvil Patel
+      organization: gsoc
+    - cisco
+    - styra
+
+  dart-authorization:
+    title: HTTP API Authorization in Dart
+    description: This integration demonstrates how to leverage OPA to perform basic HTTP API authorization in a simple Dart microservice. OPA makes it possible to provide fine-grained context-aware authorization for each REST endpoint and access method.
+    labels:
+      layer: network
+      category: application
+    software:
+    - dart
+    tutorials:
+    - https://github.com/adaptant-labs/opa-api-authz-dart/README.md
+    code:
+    - https://github.com/adaptant-labs/opa-api-authz-dart
+    inventors:
+    - adaptant
+
+  springsecurity-api:
+    title: Authorization for Java Spring Security
+    description: Spring Security provides a framework for securing Java applications.  This integration provides a simple implementation of an AccessDecisionVoter for Spring Security that uses OPA for making API authorization decisions.
+    labels:
+      layer: network
+      category: application
+    software:
+    - javaspringsecurity
+    code:
+    - https://github.com/open-policy-agent/contrib/tree/master/spring_authz  
+    tutorials:
+    - https://github.com/open-policy-agent/contrib/blob/master/spring_authz/README.md
+    inventors:
+    - styra
+
+  spinnaker-pipeline:
+    title: Spinnaker Pipeline Policy Enforcment
+    description: Spinnaker is a  Continuous Delivery and Deployment tool started by Netflix.  OPA lets you configure policies that dictate what kinds of Spinnaker pipelines developers can create.
+    labels:
+      layer: cicd
+    software:
+      - spinnaker
+    tutorials:
+    - https://docs.armory.io/spinnaker/policy_engine/
+    inventors:
+    - armory
+
+  elasticsearch-datafiltering:
+    title: Elasticsearch Data Filtering
+    description: Elasticsearch is a distributed, open source search and analytics engine.  This OPA integration lets an elasticsearch client construct queries so that the data returned by elasticsearch obeys OPA-defined policies.
+    labels:
+      layer: data
+      category: filtering
+    software:
+    - elasticsearch
+    code:
+    - https://github.com/open-policy-agent/contrib/tree/master/data_filter_elasticsearch
+    tutorials:
+    - https://github.com/open-policy-agent/contrib/blob/master/data_filter_elasticsearch/README.md
+    inventors:
+    - styra
+
+  # azure-cosmos-datafiltering:
```

Comment:
Added a comment explaining that I'm struggling to give a good description and understand exactly what the code is illustrating because there is more than one actual integration.